### DISCUSSION TITLE
插件安装失败

### DISCUSSION THREAD

**Discussion by b0y6w9** [MuiseDestiny/zotero-gpt#180] [Q&A] [answered]
这是为啥呀？
把文件拖过去里面的时候，zotero显示无法安装插件“%S”。它可能无法与该版本的 Zotero 兼容。

**Comment by MuiseDestiny** [comment, marked as answer]
xpi后缀文件为插件

**Reply by b0y6w9** [reply]
谢谢回答！已经下载成功，但是请问运行的时候又显示
![image](https://github.com/MuiseDestiny/zotero-gpt/assets/136608650/fe93bd57-1bd0-456e-9c66-478b9bb8042a)
又是怎么一回事？

**Reply by MuiseDestiny** [reply]
输入/report 回车
看下你的配置

**Reply by b0y6w9** [reply]
`api` https://api.openai.com
`secretKey` 你的k...的key
`model` gpt-3.5-turbo
`temperature` 1.0

**Reply by b0y6w9** [reply]
明白了，谢谢！
